private_gpt/ui/ui.py

```diff
@@ -418,11 +418,54 @@ def _build_ui_blocks(self) -> gr.Blocks:
                         inputs=system_prompt_input,
                     )
 
+                    def get_model_label() -> str:
```

Comment:
You are returning None in line 451, so the typing check fails with

```error: Incompatible return value type (got "None", expected "str")  [return-value]````

You can just change the return type 
from
```
-> str:
```
to
```
-> str | None:
```

Reply:
Thanks, I just made that change.